samcli/lib/docs/documentation.py

```diff
@@ -23,3 +36,27 @@ def open_docs(self):
         """
         LOG.debug(f"Launching {self.url} in a browser.")
         self.browser.launch(self.url)
+
+    def get_docs_link_for_command(self) -> str:
+        """
+        Get the documentation URL from a specific command
+
+        Returns
+        -------
+        str
+            String representing the link to be opened
+        """
+        return Documentation.load().get(self.command, LANDING_PAGE)
+
+    @staticmethod
+    def load() -> Dict[str, Any]:
+        """
+        Opens the configuration file and returns the contents
+
+        Returns
+        -------
+        Dict[str, Any]
+            A dictionary containing commands and their corresponding docs URLs
+        """
+        with open(Path(__file__).parent / DOCS_CONFIG_FILE) as f:
+            return dict(json.load(f))
```

Comment:
Thoughts about having the links as constants in code instead of a JSON file to avoid reading a file?

Reply:
We can do either. I had initially played with some different JSON structures to store the data instead of just a simple KV pair which is why I went with JSON, though I don't think reading a file here is overly expensive.